test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -467,7 +452,6 @@ Describe "TabCompletion" -Tags CI {
             $beforeTab = "\\localhost\$homeDrive\wind"
             $afterTab = "& '\\localhost\$homeDrive\Windows'"
             $res = TabExpansion2 -inputScript $beforeTab -cursorColumn $beforeTab.Length
-            $res.CompletionMatches.Count | Should BeExactly 1
```

Comment:
Maybe `Should BeGreaterThan 0` - else next line can throw with useless message.

Reply:
Can you elaborate. I want to check it for being exactly 1. If the check fails, the next line will not be executed and the It will be marked as 'Failed'.

Reply:
If $res.CompletionMatches.Count = 0 then $res.CompletionMatches[0] will throw with generic message.

Reply:
But if  $res.CompletionMatches.Count = 2 and we are using `Should BeGreaterThan 0` then the test might pass but we are getting more matches than expected.

Reply:
I wonder now - why are we removing this line with `Should BeExactly 1` if we want just 1 ?

Reply:
Correct, I confused this with the about_help test. This can be BeGreaterThan 0.

Reply:
Fixed.